internal/gcpubsub/subscribe.go

```diff
@@ -0,0 +1,98 @@
+package gcpubsub
```

Comment:
I don't love this package name, but I'm guessing if we just called it `pubsub` it'd clash with `cloud.google.com/go/pubsub`?

Reply:
exactly that, i had it as pubsub originally but ideally it needs to be unique and i couldn't think of anything better